include/deal.II/fe/mapping_q_cache.h

```diff
@@ -111,6 +125,40 @@ class MappingQCache : public MappingQGeneric<dim, spacedim>
                const typename Triangulation<dim, spacedim>::cell_iterator &)>
                &compute_points_on_cell);
 
+  /**
+   * Initialize the data cache by computing the mapping support points for all
+   * cells (on all levels) of the given triangulation and a given @p mapping
+   * and transforming these points via the function @p transformation_function.
+   *
+   * The bool @p function_describes_relative_displacement is in the default
+   * case false and indicates that the function @p transformation_function
+   * maps to absolute coordinates.If the parameter is set to true, the return
+   * value of the function is interpreted as relative deformation and the result
+   * is added to the original point.
```

Comment:
```suggestion
   * is eventually added to the original point for the definition of the support
   * points of this class.
```

Reply:
I don't understand "definition of the support points"

Reply:
'definition' is maybe the wrong word, so what about `for the support points eventually used by this class`? What I wanted to express is a reminder to the user about what happens with the result of the computation provided here.